src/main/java/de/ialistannen/lighthouse/cli/CliArguments.java

```diff
@@ -66,10 +66,13 @@ public interface CliArguments {
   @Option(names = "--bot-channel-id", description = "The channel id the bot should send updates to")
   Optional<String> botChannelId();
 
-  @Parameter(index = 0, description = "Discord webhook URL or discord bot token", paramLabel = "URL|TOKEN")
+  @Option(names = "--ntfy", description = "Use ntfy to send notifications and receive update requests. Default: false")
+  boolean ntfy();
+
+  @Parameter(index = 0, description = "Webhook URL or discord bot token", paramLabel = "URL|TOKEN")
   String webhookUrlOrToken();
```

Comment:
This can now also take a ntfy webhook URL, right? Can you add that to the description?